advanced-tools/gdb.rst

```diff
@@ -21,7 +21,7 @@ GDB 7 and later
 ===============
 
 In gdb 7, support for `extending gdb with Python
-<https://sourceware.org/gdb/current/onlinedocs/gdb/Python.html#Python>`_ was
+<https://sourceware.org/gdb/onlinedocs/gdb/Python.html#Python>`_ was
```

Comment:
We don't really need the `#Python` bit. There is an anchor on the page, but it's right at the top.

```suggestion
<https://sourceware.org/gdb/onlinedocs/gdb/Python.html>`_ was
```

Reply:
understood, thanks for the feedback I'll change it right away